tdrs-backend/tdpservice/users/migrations/0031_remove_federal_gov_stt.py

```diff
@@ -0,0 +1,27 @@
+import django.db.models.deletion
+from django.db import migrations, models
+
+def remove_federal_stt(apps, schema_editor):
+    try:
+        STT=apps.get_model('stts','STT')
+        User=apps.get_model('users','User')
+        federal_stt=STT.objects.get(name='Federal Government')
+        federal_stt_users=User.objects.filter(stt=federal_stt.id)
+
+        for user in federal_stt_users:
+            user.location_type = None
+            user.location_id = None
+    except:
+        pass
```

Comment:
Can throw debug logging in this except to get precise information on how this migration handles the fed gov user case.